core/src/executor/fetch.rs

```diff
@@ -140,6 +144,11 @@ pub async fn fetch_relation_rows<'a>(
 
             Ok(Rows::Table(stream::iter(rows)))
         }
+        TableFactor::Series { size, .. } => {
+            let rows = (1..=*size).collect::<Vec<_>>();
+            let rows = stream::iter(rows).map(|v| Ok(Row(vec![Value::I64(v)])));
+            Ok(Rows::Series(rows))
```

Comment:
Use of `collect` looks unnecessary.
It would be ok without `collect`
```rust
let rows = (1..=*size).map(|v| Ok(Row(vec![Value::I64(v)])));

Ok(Rows::Series(stream::iter(rows)))
```